src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -212,9 +214,18 @@ const PIAFormPage = () => {
         const updatedPia = await upsertAndUpdatePia({
           status: PiaStatuses.MPO_REVIEW,
         });
-        navigate(routes.PIA_RESULT, {
-          state: { result: updatedPia },
-        });
+        if (pia?.id) {
+          navigate(
+            buildDynamicPath(routes.PIA_NEXT_STEPS_EDIT, {
+              id: pia.id,
+              title: pia.title,
+            }),
+          );
+        } else {
+          navigate(routes.PIA_NEXT_STEPS_EDIT, {
+            state: { result: updatedPia },
+          });
```

Comment:
if pia.id is not there, we should not perform any action.

Maybe disable the submit button, until we have pia.id ??

Reply:
needs to be addresses more broadly.